src/AbstractFixer.php

```diff
@@ -80,6 +86,63 @@ public function supports(\SplFileInfo $file)
         return true;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            @trigger_error(
+                'Passing NULL to set default configuration is deprecated and will not be supported in 3.0, use an empty array instead.',
+                E_USER_DEPRECATED
+            );
```

Comment:
:-1: for this change. If we want to deprecate sth, I would rather split this method into 2 - one to set config, one to reset it